lib/src/models/phone_number_range.dart

```diff
@@ -0,0 +1,76 @@
+import '../../phone_numbers_parser.dart';
+
+/// Describes a contiguous range of phone numbers
+class PhoneNumberRange {
+  final PhoneNumber start;
+  final PhoneNumber end;
+
+  const PhoneNumberRange(
+    this.start,
+    this.end,
+  );
```

Comment:
yes we only ever use forward ranges.
I think it might be safer if we throw if the try to supply a forward range.
If we just swap we might cause confusion when the range is printed.